http-server-netty/src/main/java/io/micronaut/http/server/netty/configuration/NettyHttpServerConfiguration.java

```diff
@@ -1485,6 +1486,30 @@ public void setExposeDefaultRoutes(boolean exposeDefaultRoutes) {
             this.exposeDefaultRoutes = exposeDefaultRoutes;
         }
 
+        /**
+         * If {@code true} (the default), this listener will stop accepting new connections and
+         * terminate any existing ones when a graceful shutdown is initiated. By setting this to
+         * {@code false} you can ignore the graceful shutdown, e.g. to keep a management port up
+         * while the shutdown of other listeners is in progress.
+         *
+         * @return Whether to support shutting down gracefully
+         */
+        public boolean isSupportGracefulShutdown() {
```

Comment:
I would move this to `HttpServerConfiguration` and create a nested class called `GracefulShutdownConfiguration` that has enabled/disabled as well as duration, timeouts etc. other potential configuration options.

Reply:
This option is very specific to individual listeners. It exists just to exclude listeners from the graceful shutdown entirely, in order to keep up management endpoints. It is not meant as wider graceful shutdown config